master/docs/relnotes/0.9.0rc2.rst

```diff
@@ -0,0 +1,77 @@
+Release Notes for Buildbot ``0.9.0rc2``
+========================================
+
+
+The following are the release notes for Buildbot ``0.9.0rc2``.
+
+See :ref:`Upgrading to Nine` for a guide to upgrading from 0.8.x to 0.9.x
+
+
+Master
+------
+
+Features
+~~~~~~~~
+
+* add a UI button to allow to cancel the whole queue for a builder
+
+Fixes
+~~~~~
+
+* fix the UI to allow to cancel a buildrequest (:bug:`3582`)
+* Fix BitbucketPullrequestPoller change detection
+* Fix customization for template_type in email reporter
+* fix DockerLatent integration of volumes mounting
+* misc doc fixes
+* fix buildbot not booting when builder tags contains duplicates
+* forcesched: fix owner parameter when no authentication is used
+* REST: fix problem with twisted 16 error reporting
+* CORS: format errors according to API type
+* Dockerfiles fix and upgrade ubuntu to 16.04
+* Fixes #3430 Increased size of builder identifier from 20 to 50 (brings it in line to size of steps and workers in same module).
+* Fix missing VS2015 entry_points
+
+Changes for Developers
+~~~~~~~~~~~~~~~~~~~~~~
+
+Features
+~~~~~~~~
+
+Fixes
+~~~~~
+
+
+Deprecations, Removals, and Non-Compatible Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* remove repo from worker code (obsoleted by repo master source step)
+
+
+Buildslave
```

Comment:
Isn't buildslave was removed from the source tree and 0.9 and after releases?
I this there is no need for changelog for it anymore.